DentOS_Framework/DentOsTestbedLib/test/test_dent_sysctl.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# generated using file ./gen/model/dent/system/os/sysctl.yaml
+#
+# DONOT EDIT - generated by diligent bots
+
+import asyncio
+from .utils import TestDevice
+from dent_os_testbed.lib.os.sysctl import Sysctl 
+def test_that_sysctl_get(capfd):
```

Comment:
Please add 2 blank lines before `def`.

Reply:
This file is auto-generated, so it probably will be changed the next time someone generates a new API.